universum/modules/output/output.py

```diff
@@ -22,15 +22,24 @@ class Output(Module):
     terminal_driver_factory = Dependency(TerminalBasedOutput)
     html_driver_factory = Dependency(HtmlOutput)
 
+    html_log_default_name = "universum_log"
+    html_log_disabled_arg_value = "disabled"
```

Comment:
In my opinion, the value for the disabling the log should be chosen more carefully.
With current value user can run the following command that doesn't produce html log:
```
python -m universum ... --html-log disabled
```
As a user I would expect that there are no "special" undocumented file names that I cannot use for my html logs.